$NetBSD: patch-libjava_contrib_rebuild-gcj-db.in,v 1.1 2013/04/01 21:06:17 wiz Exp $

No need to use bash for this small script.

Also -print0 is not portable to solaris and perhaps others.

--- libjava/contrib/rebuild-gcj-db.in.orig	2008-07-02 13:17:54.000000000 +0000
+++ libjava/contrib/rebuild-gcj-db.in
@@ -1,4 +1,4 @@
-#!/bin/bash
+#!/bin/sh
 # rebuild-gcj-db
 
 ## Copyright (C) 2000, 2002, 2003, 2008 Free Software Foundation
@@ -16,12 +16,12 @@ for i in `@prefix@/bin/gcc --print-multi
    base=@prefix@/lib/$dirname
    dbLocation=`@prefix@/bin/gcj-dbtool -p $base`
    libdir=$base/gcj
-   if ! test -d $libdir; then
+   if test ! -d $libdir; then
       # No shared libraries here.
       continue
    fi
    dirname $dbLocation | xargs mkdir -p
    @prefix@/bin/gcj-dbtool -n $dbLocation 64
-   find $libdir -follow -name '*.db' -print0 | \
+   find $libdir -follow -name '*.db' -print | @AWK@ '{printf("%s%c", $1, 0);}'|\
       @prefix@/bin/gcj-dbtool -0 -m $dbLocation $dbLocation
 done
